### DISCUSSION TITLE
Can we get a General config to disable "Find and Replace" ?

### DISCUSSION THREAD

**Discussion by leowebguy** [craftcms/cms#14044] [Ideas] [answered]
I'd like to disable it even for Admins in production.
Thanks!

**Comment by brandonkelly** [comment, marked as answer]
Just added a new `disabledUtilities` config setting, for Craft 4.6 (2952b85316f8edfc3a1af85a4469897974ea9b06). With that, you will be able to fully disable any utilities by their ID:

```php
// config/general.php

->disabledUtilities([
    'updates',
    'find-replace',
])
```

```sh
# .env

CRAFT_DISABLED_UTILITIES=updates,find-replace
```

**Reply by brandonkelly** [reply]
Craft 4.6.0 is out with that new setting.

**Reply by leowebguy** [reply]
Thanks Brandon, ya da best.
